src/Symfony/Component/Encryption/KeyInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Encryption;
+
+/**
+ * A key for a specific user and specific Encryption implementation. Keys cannot
+ * be shared between Encryption implementations.
+ *
+ * A key is always serializable.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+interface KeyInterface extends \Serializable
```

Comment:
please don't enforce the usage of `\Serializable`. It is not the only way to make a PHP serializable (and it is the worse one actually)

Reply:
Could you help me undertand what I should use instead?

I dont want magic methods in the interface, do I?

Reply:
I would keep it only in the phpdoc, and let each implementation decide what they want to use

Reply:
Done, thank you